src/common/components/HelperText.js

```diff
@@ -132,14 +135,26 @@ const HelperText = props => {
 
   return (
     <>
-      <IconButton
-        size="small"
-        ref={anchorEl}
-        aria-label={t('form.helper_text_info_label', { label })}
-        onClick={handleClick}
-      >
-        <InfoIcon fontSize="1rem" />
-      </IconButton>
+      {showLabel ? (
```

Comment:
question: when is `showLabel` true?

Reply:
@paulschreiber It is true for this change, so in this case we need to show not just the helper text icon but also a label like this:
<img width="233" alt="Screenshot 2023-08-28 at 3 37 31 PM" src="https://github.com/techmatters/terraso-web-client/assets/1316782/817f891b-4006-489d-bea7-c555c49110f3">
For most cases it is just an icon button, like this:
<img width="197" alt="Screenshot 2023-08-28 at 3 38 05 PM" src="https://github.com/techmatters/terraso-web-client/assets/1316782/faea014b-f2c3-40ee-9190-813292d6bc4c">

I think it looks similar but in the first case you can also click on the text to open the helper text, and the text is specifically for the helper button